<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>直方图</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="10,0" rowminimumheight="10,0">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="originalImg_label">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="resultImg_label">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>源图像</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>处理结果</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSpinBox" name="spinBox"/>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider">
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QSpinBox" name="spinBox_2"/>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="on_openfile"/>
    <addaction name="on_restore"/>
    <addaction name="ac_clear"/>
    <addaction name="ac_save"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>图像处理</string>
    </property>
    <addaction name="ac_histogramRGB"/>
    <addaction name="ac_histogramGray"/>
    <addaction name="ac_threshold"/>
    <addaction name="ac_test"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="on_openfile">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
  </action>
  <action name="on_restore">
   <property name="text">
    <string>还原</string>
   </property>
   <property name="toolTip">
    <string>还原图像</string>
   </property>
  </action>
  <action name="ac_clear">
   <property name="text">
    <string>清除</string>
   </property>
   <property name="toolTip">
    <string>清除图像</string>
   </property>
  </action>
  <action name="ac_save">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存图像</string>
   </property>
  </action>
  <action name="ac_histogramRGB">
   <property name="text">
    <string>三通道直方图</string>
   </property>
  </action>
  <action name="ac_histogramGray">
   <property name="text">
    <string>灰度直方图</string>
   </property>
   <property name="toolTip">
    <string>灰度直方图</string>
   </property>
  </action>
  <action name="ac_threshold">
   <property name="text">
    <string>阈值分割</string>
   </property>
   <property name="toolTip">
    <string>阈值分割</string>
   </property>
  </action>
  <action name="ac_test">
   <property name="text">
    <string>滑条实验</string>
   </property>
   <property name="toolTip">
    <string>实验程序</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
